code/game/objects/structures/barsigns.dm

```diff
@@ -111,6 +111,11 @@
 	icon = "slipperyshots"
 	desc = "Slippery slope to drunkeness with our shots!"
 
+/datum/barsign/thegreytide
+	name = "The Grey Tide"
+	icon = "thegreytide"
+	desc = "Take a break from toolboxing and enjoy a lazy beer!"
```

Comment:
Requesting an edit to "Abandon your toolboxing ways" from "Take a break from toolboxing" since as has been vigorously argued, the station is emotionally sensitive to the bar display and you can use it to manipulate them into griefing